submissions/dev-adventure/index.js

```diff
@@ -0,0 +1,77 @@
+const net = require('net');
+
+const startCheck = async () => {
+  try {
+    const arg = process.argv.slice(2);
+    if (!arg || !arg.length) {
+      process.stdout.write(
+        '--host <domain name or ip what you want to check>' +
+          ' is required arguments.\n' +
+          '--help for get description.'
+      );
+      process.exit(1);
+    }
+    if (arg[0] === '--help') {
+      process.stdout.write(
+        '--host <domain name or ip what you want to check> - required argument.\n' +
+          '--ports <start_port-end_port> - ports range from/to.\n' +
+          '--help - get help.'
+      );
```

Comment:
I guess you need to exit here. Otherwise the program will show help and continue running.

Reply:
Thanks